homeassistant/components/ios.py

```diff
@@ -245,34 +243,21 @@ def setup(hass, config):
     if CONFIG_FILE == {}:
         CONFIG_FILE[ATTR_DEVICES] = {}
 
-    device_tracker = loader.get_component("device_tracker")
-    if device_tracker.DOMAIN not in hass.config.components:
-        device_tracker.setup(hass, {})
-        # Need this to enable requirements checking in the app.
-        hass.config.components.append(device_tracker.DOMAIN)
-
-    if "notify.ios" not in hass.config.components:
-        notify = loader.get_component("notify.ios")
-        notify.get_service(hass, {})
-        # Need this to enable requirements checking in the app.
-        if NotifyDomain not in hass.config.components:
-            hass.config.components.append(NotifyDomain)
-
-    zeroconf = loader.get_component("zeroconf")
-    if zeroconf.DOMAIN not in hass.config.components:
-        zeroconf.setup(hass, config)
-        # Need this to enable requirements checking in the app.
-        hass.config.components.append(zeroconf.DOMAIN)
+    bootstrap.setup_component(hass, "device_tracker")
```

Comment:
There is no `ios` platform for `device_tracker`. It only needs to be loaded so that the iOS app can fire `device_tracker/see` events. There is an upcoming (0.32) piece to persist location across reboots of HASS.